packages/@yoda/util/retries.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+module.exports.retries = retries
+
+function retries (maxRetry, thisArg, action) {
+  var retry = maxRetry || 1
+  var argLen = arguments.length
+  var lastly = false
+  if (argLen === 2) {
+    action = thisArg
+  }
+  var tryAction = function () {
+    retry--
+    if (retry <= 0) {
+      lastly = true
+    }
+    if (argLen > 2) {
+      action.call(thisArg, tryAction, lastly)
+    } else {
+      action(tryAction, lastly)
```

Comment:
Does it better to return the return value of action here?

Reply:
yes, I forgot it.